### PR TITLE
YLP-681 Snackbar alert hook

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/components/routes.tsx [hunk 1]
```diff
@@ -57,7 +60,9 @@ export const PrivateRoute = (props: RouteProps): JSX.Element => {
   return isLoggedIn() ? (
     <ThemeProvider theme={mainTheme}>
       <CssBaseline />
-      <Route {...props} />
+      <SnackbarContextProvider context={DefaultSnackbarContext}>
```
Comment (Nas84) [inline packages/yourloops/components/routes.tsx:63]: i will have put it in app.ts to avoid twice declarations, however we could leave it this way

**Review by Nas84 — APPROVED** [review]
LGTM

File: packages/yourloops/components/routes.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/components/routes.tsx:63]: It was my original idea. But the Theme is different, and all the colors in the rest of the UI change when an alert is displayed.
So to make it quick I duplicate this.

Comment (Nas84) [reply packages/yourloops/components/routes.tsx:63]: oki it is fine for me

**Pull request merged**